googletest/include/gtest/internal/gtest-port.h

```diff
@@ -434,10 +434,7 @@ typedef struct _RTL_CRITICAL_SECTION GTEST_CRITICAL_SECTION;
 # endif
 #endif
 
-#if GTEST_USES_PCRE
```

Comment:
Why are we removing GTEST_USES_PCRE? We should treat it just like any other currently undefined macros.